lib/track.js

```diff
@@ -140,12 +145,38 @@ Track.prototype._send = function(type, name, props) {
         pageTitle: document.title,
         referrer: document.referrer || undefined,
         clientEventTimeMilliseconds:  Date.now ? Date.now() : +new Date(),
+        incognitoDetected: incognitoDetected,
```

Comment:
Think we could simplify it to `incognitoDetected, ` only

```suggestion
        incognitoDetected,
```

Reply:
Unfortunately this style is needed since this repo requires [IE-compatible JavaScript](https://github.com/unravelin/ravelinjs/blob/v1/CONTRIBUTING.md#6-write-ie-compatible-code-in-lib), and object shorthand isn't supported in IE.